config/salt/config/nginx/default

```diff
@@ -11,8 +11,14 @@ server {
 
         client_max_body_size 50M;
 
-        if ( !-d /srv/www/$root ) {
-                set $root 'default';
+        # Thunderhead
+        if ( -d /srv/www/$root/public ) {
```

Comment:
Need to ask @joehoyle how best to do that, seeing as the repos are designed to be web roots it would have to be some addition to the provisioner that checks for additional files in the repo under a non web accessible folder eg.

```
.config/
.config/nginx.conf   # additional location blocks etc...
.config/app.sls        # additional provisioning steps eg. php5-ldap
```

I had something similar but the provisioner would delete the folder afterwards just to be extra safe.